python/mlc_chat/cli/compile.py

```diff
@@ -101,6 +101,12 @@ def _check_system_lib_prefix(prefix: str) -> str:
         default="",
         help=HELP["overrides"] + ' (default: "%(default)s")',
     )
+    parser.add_argument(
+        "--debug-dump-path",
+        type=str,
```

Comment:
As part of the new convention, we prefer always using `pathlib.Path` to represent paths to directories/files to avoid mishandlings that have happened in the past. See `_parse_output` in the same file for more details